examples/channels-for-workers/channels_workers.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/io;
+//Defines a channel that accepts xml messages.
+channel<xml> xmlChn;
+
+public function main(string... args) {
```

Comment:
Since we are not using `args` within the `main` function we can remove `args` and define the `main` function without any parameters.